src/stable_browser.ts

```
@@ -3088,6 +3097,96 @@ class StableBrowser {
+            if (results.length === 0) {
+              throw new Error(`Element not found in cell area`);
+            }
+            state.element = results[0];
+            await results[0].click();
+          }
+          break;
+        case "hover+click":
+          if (!options.css) {
+            throw new Error("css is not defined");
+          }
+          const results = await findElementsInArea(options.css, cellArea, this.page, this, options);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)